### ISSUE TITLE
Provide better info from dicomtocsv for files missing from DICOMDIR.

### ISSUE THREAD

**Issue by dgobbi** [dgobbi/vtk-dicom#124]
Sometimes a DICOMDIR or an sql file will list files that have been removed.  Currently, dicomtocsv simply reports the expected filename, but since the filename is only a number, that doesn't provide any information about the importance of the file.

Since dicomtocsv has info from the DICOMDIR, it should report (at the very least) PatientID, StudyDescription, SeriesNumber, InstanceNumber, SOPClassUID, and SOPInstanceUID.

If I wanted to get fancy, I could amalgamate the information for each series, and report series-level information followed the the specific instances that are missing.

**Comment by dgobbi** [comment]
I've pushed 663bea19 and 92d6ff81, which print out the StudyInstanceUID, SeriesInstanceUID, PatientID, StudyDate, StudyTime, StudyID, SeriesNumber, and InstanceNumber.  I decided not to print the StudyDescription because it is long and might not be ASCII (ditto for the PatientName).  Also, I decided that the StudyID, StudyInstanceUID and SeriesInstanceUID would be more useful for finding the missing file than its own InstanceUID ad ClassUID.  And I want to keep the error message short.

I'm not closing this yet, because I want to extend this feature to the other command-line programs that use vtkDICOMDirectory: dicompull, dicomdump, and dicomfind.
